concourse/resources/defaults.mako

```diff
@@ -17,6 +17,22 @@ credentials = registry_cfg.credentials()
         repository: eu.gcr.io/gardener-project/cc/job-image
         tag: "1.44.0"
 </%def>
+<%def name='task_image_defaults(registry_cfg, indent=0)'
```

Comment:
I do not like that this duplicates the code block immediately above. should there not rather be a function that accepts credentials and image reference ('repository' in the words of concouse's image resource).
then, the default image might be created by another function that presets those arguments, whereas the same function could be used to create custom (non-default) image resources